src/components/pill-selector-dropdown/PillSelector.js

```diff
@@ -189,6 +198,8 @@ class PillSelector extends React.Component<Props, State> {
             'aria-errormessage': this.errorMessageID,
         };
 
+        const PillComponent = showRoundedPills ? RoundPill : Pill;
```

Comment:
It seems like `RoundPill` could be a prop on a `Pill` component, do they need to be separate components?